### ISSUE TITLE
Not working without internet

### ISSUE THREAD

**Issue by Josema** [jaridmargolin/inspect-process#18]
Not sure if is my computer, but if i have the wifi disabled the inspector is never loaded. Chrome is opened but the debugger keeps spinning like if were loading something.

**Comment by jaridmargolin** [comment]
Good find @josema! Looks like this might be the same issue surfacing in #8.

**Comment by jaridmargolin** [comment]
I believe this bug may lie with chrome as the same bug also surfaces when manually running `node --inspect` with "wifi disabled".

```
1. Connect to network
2. Run inspect --debug-brk script.js
3. Enjoy debugging nirvana
```

```
1. Disconnect from network
2. Run inspect --debug-brk script.js
3. Stare at blank window
```

A few days ago I also stumbled on [this tweet](https://twitter.com/samzhaocn/status/795605031655378945). Didn't think much of at the time, but now it seems fairly relevant.

---

@paulirish - Any chance you could give this a quick set of 👀? I searched for existing chromium bugs but was unable to find anything.

**Comment by paulirish** [comment]
@Josema are you on windows?

**Comment by eugeneo** [comment]
The problem might be with "localhost" not resolving to 127.0.0.1 on some Windows setup (I was not able to reproduce it - but now I think that having no network might be the cause).

I updated Node 7.0 to prompt to use 127.0.0.1 instead of localhost. This module could try replacing localhost with 127.0.0.1 when starting the devtools.

**Comment by jaridmargolin** [comment]
@eugeneo @paulirish  I am seeing the issue and on OSX:

<img src="https://cldup.com/NzxLl0YBOa.gif" width="100%">

```
os: OSX El Capitan 10.11.1
node version: v7.0.0
chrome version:  54.0.2840.87
```

**Comment by Josema** [comment]
@paulirish OSX El Capitan, Chrome Version 54.0.2840.87 (64-bit), node v6.3.1

**Comment by eugeneo** [comment]
Oh, I did not recognize you were using older Node version. Node 6.x currently requires specific devtools version, that is downloaded from the cloud (hence "remote" in the URL string).

Node 7.x and on will be exposing stable protocol and will be able to be debugged with the devtools version bundled with the Chrome.

**Comment by jaridmargolin** [comment]
@eugeneo  Ahhhh ok that makes complete sense. Yea seeing it is still required in v7.0.0 as well (https://github.com/nodejs/node/blob/v7.0.0/src/inspector_agent.cc#L51). Appears to be fixed in v7.1.0.

Thanks for looking into it!

---

Wonder if it is possible, or even worth it, to download the remote debugger when installing this package.

**Comment by jaridmargolin** [comment]
Added "Internet Connectivity" section under "Known Issues" in README.
